std_msgs/String tf_id
std_msgs/Int32 image_x
std_msgs/Int32 image_y
std_msgs/Int32 image_height
std_msgs/Int32 image_width
